### ISSUE TITLE
[0.80] Cherry-pick stabilize custom accessibility fix

### ISSUE THREAD

**Issue by jonmajorc** [reactwg/react-native-releases#1193]
### Target Branch

0.80

### Link to commit or PR to be picked

https://github.com/facebook/react-native/commit/626568f9a3f956a52f6c55df1dc3bc5cd017e353

### Description

This commit is essential for users who rely on accessibility actions and should be cherry-picked back to improve accessibility.
